File: Selection statement/Question5/question5.java
import java.util.Scanner;

public class question5 {

    static void grade(int score) {
        if (score > 70) {
            System.out.println("You got an A");
        } else if (score > 60 && score <= 70) {
            System.out.println("You got a B");
        } else if (score > 45 && score <= 60) {
            System.out.println("You got a B+");
        } else if (score > 35 && score <= 45) {
            System.out.println("You got a C");
        } else if (score < 35) {
            System.out.println("You are fail!");
        } else {
            System.out.println("Invalid score");
        }
    }

    public static void mian(String[] args) {
        try (Scanner input = new Scanner(System.in)) {
            System.out.println("Enter your score: ");
            int score = input.nextInt();
            grade(score);
        }
    }
}

File: Selection statement/Question5/question5.c
#include <stdio.h>

int grade(int score)
{
    if (score >= 70)
    {
        printf("You got an A.\n");
    }
    else if (score >= 60 && score < 70)
    {
        printf("You got a B+.\n");
    }
    else if (score >= 45 && score < 60)
    {
        printf("You got a B.\n");
    }
    else if (score >= 35 && score < 45)
    {
        printf("You got a C.\n");
    }
    else
    {
        printf("You are fail.\n");
    }
}

int main()
{
    int score;
    printf("Enter your score: ");
    scanf("%d", &score);
    grade(score);
}